code/obj/mining.dm

```diff
@@ -1961,6 +1961,7 @@ obj/item/clothing/gloves/concussive
 		for (var/supertype in allowed_supertypes)
 			for (var/subtype in typesof(supertype))
 				allowed_types[subtype] = 1
+		allowed_types -= /obj/storage/closet/flock
```

Comment:
Pretty sure this is redundant, there's code to prevent cargo teleportation of flock closets elsewhere

Reply:
That actually didn't work, it needed to be put here too